examples/cts1-nft/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "cts1-nft"
+version = "0.1.0"
+authors = ["Concordium <developers@concordium.com>"]
+edition = "2018"
+license = "MPL-2.0"
+
+[features]
+default = ["std"]
+std = ["concordium-std/std", "concordium-cts1/std"]
+
+[dependencies]
+concordium-std = {path = "../../concordium-std", default-features = false}
+concordium-cts1 = {path = "../../concordium-cts1", default-features = false}
+
+[lib]
+crate-type=["cdylib", "rlib"]
+
+[profile.release]
+opt-level = "s"
```

Comment:
Is `s` here better than `z`? Also have you tried codegen-units=1?

Reply:
yes, "s" is the better option here. 

I have just tried setting codegen-units to 1 and it saves another 1.6 kB